src/L2dotNET/Network/GamePacketHandler.cs

```diff
@@ -40,42 +40,87 @@ static GamePacketHandler()
             ClientPackets.TryAdd(0x14, typeof(RequestUseItem));
             ClientPackets.TryAdd(0x16, typeof(RequestAddTradeItem));
             ClientPackets.TryAdd(0x17, typeof(RequestTradeDone));
-            ClientPackets.TryAdd(0x1A, typeof(RequestStartTrade));
+            ClientPackets.TryAdd(0x1a, typeof(RequestStartTrade));
             ClientPackets.TryAdd(0x1b, typeof(RequestSocialAction));
-            ClientPackets.TryAdd(0x1e, typeof(RequestSellItem));
+            //ClientPackets.TryAdd(0x1C, typeof(RequestChangeMoveType)); in L2J is RequestChangeMoveType()
+            //ClientPackets.TryAdd(0x1D, typeof(RequestChangeWaitType)); in L2J is RequestChangeWaitType()
+            ClientPackets.TryAdd(0x1e, typeof(RequestSellItem)); 
             ClientPackets.TryAdd(0x1f, typeof(RequestBuyItem));
             ClientPackets.TryAdd(0x20, typeof(RequestLinkHtml));
-            ClientPackets.TryAdd(0x20, typeof(RequestLinkHtml));
             ClientPackets.TryAdd(0x21, typeof(RequestBypassToServer));
+            //ClientPackets.TryAdd(0x22, typeof(RequestBBSwrite)); in L2J is RequestBBSwrite()
+            //ClientPackets.TryAdd(0x23, typeof(DummyPacket)); in L2J is DummyPacket()
+            //ClientPackets.TryAdd(0x24, typeof(RequestJoinPledge)); in L2J is RequestJoinPledge()
+            //ClientPackets.TryAdd(0x25, typeof(RequestAnswerJoinPledge)); in L2J is RequestAnswerJoinPledge()
             ClientPackets.TryAdd(0x26, typeof(RequestWithdrawalPledge));
+            //ClientPackets.TryAdd(0x27, typeof(RequestOustPledgeMember)); in L2J is RequestOustPledgeMember()
+            //ClientPackets.TryAdd(0x28, typeof(RequestDismissPledge)); in L2J is RequestDismissPledge()
             ClientPackets.TryAdd(0x29, typeof(RequestJoinParty));
             ClientPackets.TryAdd(0x2a, typeof(RequestAnswerJoinParty));
             ClientPackets.TryAdd(0x2b, typeof(RequestWithDrawalParty));
             ClientPackets.TryAdd(0x2c, typeof(RequestOustPartyMember));
+            //ClientPackets.TryAdd(0x2D, typeof(RequestDismissParty)); in L2J is RequestDismissParty()
+            //ClientPackets.TryAdd(0x2E, typeof(DummyPacket)); in L2J is DummyPacket()
```

Comment:
there are also 2 more references of this in the lines below. Please keep only the valid one.

Reply:
Dummy i will review what is it.

and i put the method beside of atcual method use to compare what is the error